backend/tests/test_vote.py

```diff
@@ -0,0 +1,359 @@
+import pytest
+from mongoengine import connect
+import app
+import test_data
+import model
+import json
+import copy
+
+
+@pytest.fixture
+def db():
+    connect('mongoenginetest', host='mongomock://localhost')
+    model.Item.objects().delete()
+    yield None
+    model.Item.objects().delete()
+
```

Comment:
Looks like this kind of helper util functions are copy-pasted in multiple files, maybe move this into a test_util? I could be a little bit bike-shedding here.

Reply:
I discovered that pytest has a special conftest.py file for sharing fixtures, so thanks for the suggestion!